singlecellmultiomics/bamProcessing/bamToCountTable.py

```diff
@@ -615,6 +620,11 @@ def create_count_table(args, return_df=False):
         '-head',
         type=int,
         help='Run the algorithm only on the first N reads to check if the result looks like what you expect.')
+    argparser.add_argument(
+        '-bulk',
+        type=bool,
+        default=False,
+        help='Set True if making a count table for bulkseq data.')
```

Comment:
I would try to more clearly state the behavior and add to the help description that this performs a sum over all cells.

Reply:
Changed it thanks,